### DISCUSSION TITLE
Why can't i decrypt in place while ignoring MAC

### DISCUSSION THREAD

**Discussion by jgilfoil** [getsops/sops#942] [Q&A] [answered]
So I have an encrypted file: [`secret.sops.yaml`](https://github.com/jgilfoil/k8s-gitops/blob/main/cluster/apps/flux-system/notifications/discord/secret.sops.yaml) and when i try and decrypt it in place I get the following:

```
$sops --decrypt --in-place secret.sops.yaml
MAC mismatch. File has <signature>, computed <different signature>
```
Ok, fine, i don't know why it's doing this, but I have found that if i decrypt the file by doing `sops --ignore-mac secret.sops.yaml` it opens in the editor just fine.

However if I try and add `--ignore-mac` to my  `sops --decrypt --in-place` command, it won't decrypt, and still gives me the MAC mistmatch error.

Why doesn't `--ignore-mac` work on --decrypt --inplace and how can I get around this?

**Comment by joegraviton** [comment, marked as answer]
workaround:
```
sops --decrypt --ignore-mac secret.sops.yaml > tmp.yaml
vim tmp.yaml  # edit it
sops --encrypt tmp.yaml > secret.sops.yaml
```

**Reply by jgilfoil** [reply]
that's a great idea, thanks! For whatever reason, this stopped happening to me lately, i don't know what changed or what i did, but probably a few reboots and updates on that Ubuntu machine since then.
